src/pip/_internal/commands/install.py

```diff
@@ -206,6 +216,11 @@ def run(self, options, args):
             install_options.append('--user')
             install_options.append('--prefix=')
 
+        # If --user or --global is not passed, use some heuristic to determine
```

Comment:
doesnt this have to go above to ensure the sanity check and the install option fixup?

Reply:
Yep.

Note to self: Be more careful with your merges.